### PR TITLE
Merge sensor-list updates

### PR DESCRIPTION
**Pull request by lvdheever**
This brings in the camv19 changes of sensor-list to master.
HideWarn checkbox and only filter on Search box on submit (pressing enter)

(@ajoubertza please note as you are fiddling with sensor-list)

### CODE REVIEW

File: app/sensor-list/sensor-list.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ <form novalidate name="frm1" autocomplete="off" ng-submit="vm.searchFilter=searchText">
```
Comment (ajoubertza) [inline app/sensor-list/sensor-list.html]: As we noticed, this affects the scrolling of the layout - components, proxy and sensor list header are no longer static.  Would be good to fix this.

File: app/sensor-list/sensor-list.js [hunk 2]
```diff
@@ -273,16 +275,21 @@
...
+            return show
         };
 
         vm.updateURL = function () {
-            $state.go('sensor-list', {
-                component: vm.resourceSensorsBeingDisplayed? vm.resourceSensorsBeingDisplayed: null,
-                filter: vm.searchFilter? vm.searchFilter: null,
-                hideNominal: vm.hideNominalSensors? 'true': null},
-                { notify: false, reload: false });
+            if (!vm.noSensorUpdates) {
```
Comment (ajoubertza) [inline app/sensor-list/sensor-list.js]: What causes this variable to change?

File: app/sensor-list/sensor-list.html (hunk 1 above)
Comment (lvdheever) [reply app/sensor-list/sensor-list.html]: Yes, I added this to the list of KatGUI fixes.

File: app/sensor-list/sensor-list.js (hunk 2 above)
Comment (lvdheever) [reply app/sensor-list/sensor-list.js]: Oops - this is a remnant of a previous attempt to have a checkbox to start/stop sensor udpates, which proved to be more cumbersome than the "Enter" approach. Will remove.

**Pull request merged**